.github/workflows/desktop.yml

```diff
@@ -17,30 +17,31 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-latest]
         build_type: ["Release", "Debug"]
         architecture: ["x64", "x86"]
-        msvc_runtime: ["static", "dynamic"]
         python_version: [3.7]
         include:
         - os: windows-latest
-          vcpkg_triplet_suffix: "windows-static"
+          build_type: "Release"
+          architecture: "x64"
+          msvc_runtime: "static"
+          vcpkg_triplet: "windows-static"
```

Comment:
I think you meant vcpkg_triplet_suffix here.

Reply:
Whoops, good catch.